data/registers/canfd.yaml

```diff
@@ -0,0 +1,3284 @@
+    bit_offset: 12
+    bit_size: 1
+  - name: ND45
+    description: New Data RX Buffer 45 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 13
+    bit_size: 1
+  - name: ND46
+    description: New Data RX Buffer 46 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 14
+    bit_size: 1
+  - name: ND47
+    description: New Data RX Buffer 47 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 15
+    bit_size: 1
+  - name: ND48
+    description: New Data RX Buffer 48 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 16
+    bit_size: 1
+  - name: ND49
+    description: New Data RX Buffer 49 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 17
+    bit_size: 1
+  - name: ND50
+    description: New Data RX Buffer 50 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 18
+    bit_size: 1
+  - name: ND51
+    description: New Data RX Buffer 51 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 19
+    bit_size: 1
+  - name: ND52
+    description: New Data RX Buffer 52 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 20
+    bit_size: 1
+  - name: ND53
+    description: New Data RX Buffer 53 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 21
+    bit_size: 1
+  - name: ND54
+    description: New Data RX Buffer 54 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 22
+    bit_size: 1
+  - name: ND55
+    description: New Data RX Buffer 55 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 23
+    bit_size: 1
+  - name: ND56
+    description: New Data RX Buffer 56 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 24
+    bit_size: 1
+  - name: ND57
+    description: New Data RX Buffer 57 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 25
+    bit_size: 1
+  - name: ND58
+    description: New Data RX Buffer 58 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 26
+    bit_size: 1
+  - name: ND59
+    description: New Data RX Buffer 59 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 27
+    bit_size: 1
+  - name: ND60
+    description: New Data RX Buffer 60 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 28
+    bit_size: 1
+  - name: ND61
+    description: New Data RX Buffer 61 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 29
+    bit_size: 1
+  - name: ND62
+    description: New Data RX Buffer 62 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 30
+    bit_size: 1
+  - name: ND63
+    description: New Data RX Buffer 63 0 Rx Buffer not updated 1 Rx Buffer updated from new message.
+    bit_offset: 31
+    bit_size: 1
```

Comment:
NDAT1 and NDAT2 have the same layout, so these two fields can probably be merged (with `check: Layout`)